static/index.html

```diff
@@ -3,8 +3,10 @@
 <head>
     <title>Easy Phi - test measurement equipment platform</title>
     <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+    <link rel="stylesheet" type="text/css" href="http://yui.yahooapis.com/3.18.1/build/cssreset/cssreset-min.css">
     <link rel="stylesheet" href="css/style.css" type="text/css" media="screen, projection">
-    <link rel="stylesheet" href="css/jquery-ui.min.css" type="text/css" media="screen, projection">
```

Comment:
previous local hosted theme was mising some image files and caused a lot of 404 errors.
This solution is temporary, ultimate goal is to get rid of  jQueryUI completely